lib/hades/mentorships/mentoree.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Hades.Mentorships.Mentoree do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Hades.Mentorships.Mentoree
+
+  @valid_attrs ~w(is_active is_minority)a
+
+  schema "mentorees" do
+    field :is_active, :boolean, default: false
+    field :is_minority, :boolean, default: false
+    belongs_to :user, Hades.Accounts.User
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(%Mentoree{} = mentoree, attrs) do
+    mentoree
+    |> cast(attrs, @valid_attrs)
+    |> validate_required(@valid_attrs)
```

Comment:
Perhaps we should require `user_id` here and validate that it will not appear in more than one `mentoree` record using the `unique_constraint/1`.

Reply:
Thanks!
Nice, good idea! 👍

Reply:
We should actually do the same for `Mentor` 😛